lib/plugins/aws/invoke.js

```diff
@@ -106,25 +107,28 @@ class AwsInvoke {
     if (invocationReply.Payload) {
       const response = JSON.parse(invocationReply.Payload);
 
-      this.consoleLog(color(JSON.stringify(response, null, 4)));
+      legacy.consoleLog(color(JSON.stringify(response, null, 4)));
+      writeText(JSON.stringify(response, null, 4));
     }
 
     if (invocationReply.LogResult) {
-      this.consoleLog(
+      legacy.consoleLog(
         chalk.gray('--------------------------------------------------------------------')
       );
+      writeText(
```

Comment:
I've decided to keep this separator in similar manner as it was in previous implementation